### DISCUSSION TITLE
The problem of v-menu, v-select in Vuetify 3 and Nuxt 3

### DISCUSSION THREAD

**Discussion by nikkon82** [vuetifyjs/vuetify#16057] [Q&A] [answered]
When using v-menu or v-select in Nuxt 3 + Vuetify 3 the menu or drop-down list "flies away" to the upper left corner of the screen. In the console, a warning and an error:

[Vue Warning]: Unhandled error during observer callback execution

Error type: _ref2 is not repeatable

Put a clean vuetify 3 - it works fine
What could be the problem with Vuetify 3 + Nuxt 3?

**Comment by jacekkarczmarczyk** [comment, marked as answer]
https://github.com/vuetifyjs/vuetify/issues/16051

**Reply by nikkon82** [reply]
Thanks
